clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -313,7 +313,7 @@ public void onFailure(RuntimeException e) {
                 metadata());
 
         // create the request for the coordinator
-        log.debug("Issuing request ({}: {}) to coordinator {}", ApiKeys.JOIN_GROUP, request, this.coordinator.id());
+        log.debug("Sending JoinGroup ({}) to coordinator {}", request, this.coordinator.id());
```

Comment:
Do we really need two log statements so near to each other? I guess the second one also includes the co-ordinator id. Would it be too confusing to include it in the previous log statement?